lib/core/src/Ouroboros/Network/Client/Wallet.hs

```diff
@@ -490,8 +490,9 @@ localStateQuery queue =
     clientStIdle
         :: m (LSQ.ClientStIdle block (Point block) (Query block) m Void)
     clientStIdle = awaitNextCmd <&> \case
-        CmdQueryLocalState pt query respond ->
-            LSQ.SendMsgAcquire pt (clientStAcquiring query respond)
+        CmdQueryLocalState _pt query respond ->
+            -- fixme: ADP-672 not sure why it's Maybe Point now
```

Comment:
I think `Nothing` now means "use the current tip"

Reply:
Reverted with rationale -> b2388bd

Reply:
> Allowing the node to decide what the current tip is, would solve ADP-647, but it could also cause race conditions from queries potentially being run on different points.
> 
> We could still leverage Nothing in #2449.

OK good rationale.